### ISSUE TITLE
[Feature] Interactive Kanban - Drag, Drop

### ISSUE THREAD

**Issue by isosphere** [hkgnp/logseq-kanban-plugin#31]
The ability to drag and drop cards is a big deal for me; IMO it's the whole reason to Kanban!

Our upstream dependency [react-kanban](https://github.com/asseinfo/react-kanban) supports interactive boards[^1], but we'll have to add code that will modify the LogSeq graph according to actions taken by the user. 

[^1]: though it should be acknowledged that the package is abandoned

I am attempting to implement this feature on my own, but I'd be happy for any tips or discussion here. I'll open a draft PR if I have any kind of progress so that other interested parties can collaborate on the feature. 

I'm using the [react-kanban demo](https://codesandbox.io/s/react-kanban-demo-nvjp3?file=/src/index.js) as a guide.

**Comment by isosphere** [comment]
I've made an initial attempt that matches the `react-kanban` demo but the event handler doesn't get triggered on drag. I've deleted the default logseq event handlers via the developer tools as a test but it had no effect. 

The https://github.com/cannibalox/logtools project, when installed, enables showing a block as a set of kanban columns using CSS trickery. It has the added benefit of not requiring additional graph editing. It doesn't look as good, but it works. 

I'd prefer to use this plugin but I think I'd need some help getting the modifications working.  The current state of my attempt is here: https://github.com/isosphere/logseq-kanban-plugin/tree/drag-drop
